frontend/src/routes/Governance/policies/policy-details/PolicyDetailsResults.tsx

```diff
@@ -48,6 +48,15 @@ function getTemplateDetailURL(item: ResultsTableData) {
   })
 }
 
+function templateExists(msg: string): boolean {
+  return !(
```

Comment:
Could you review this @JustinKuli @dhaiducek

Reply:
Looks like the "Failed to create policy template" case catches all of the strange situations I could think up. :+1: